common/web/gesture-recognizer/tsconfig.json

```diff
@@ -5,7 +5,17 @@
     "lib": [ "dom", "es6" ],
     "outDir": "./build/obj/",
     "rootDir": "./src/engine",
-    "tsBuildInfoFile": "./build/obj/tsconfig.tsbuildinfo"
+    "tsBuildInfoFile": "./build/obj/tsconfig.tsbuildinfo",
+
+    // TODO: These override ../tsconfig.base.json settings, and so should be removed if possible,
+    //       but existing code in web/ breaks some of these settings
+    //
+    // At present, the code actually compiles without them... but the TS tests are a different matter.
```

Comment:
If these are only needed for the tests, shouldn't we move these definitions to `src/test/tsconfig.json` ?

Reply:
They'll be eliminated entirely by the end of this PR chain (#11464), so I didn't bother.